.github/ISSUE_TEMPLATE/roadmap-item.md

```diff
@@ -0,0 +1,61 @@
+---
+name: 'Roadmap Item'
+about: Create roadmap item for this project
+title: Roadmap Item =description=
+labels: kind/roadmap
+assignees: ''
+
+---
+
+# O3DE Roadmap Item Template
+
+### When using this template, you do not have to fill out every question below. The questions are there for guidance.
+
+This roadmap item template should be used for any feature that shows in the O3DE public roadmap. The issue communicates the initiative and vision of the SIG.
+
+# ----- DELETE EVERYTHING FROM THE TOP TO THE SUMMARY LINE BELOW WHEN USING TEMPLATE ----- #
+
+### Summary:
+Single paragraph explanation of the roadmap item
+
+### What is the relevance of this feature?
+What problems does it solves? Why is this important? What are the use cases? What will it do once completed?
+
+### Feature design description:
+- Explain the design of the feature with enough detail that someone familiar with the environment and framework can understand the concept and explain it to others. 
+- It should include at least one end-to-end example of how a developer will use it along with specific details, including outlying use cases. 
+
+- If there is any new terminology, it should be defined here.
+
+### Technical design description:
+- Explain the technical portion of the work in enough detail that members can implement the feature. 
+
+- Explain any API or process changes required to implement this feature
+
+- This section should relate to the feature design description by reference and explain in greater detail how it makes the feature design examples work.
+
+- This should also provide detailed information on compatibility with different hardware platforms.
+
+### What are the advantages of the feature?
+- Explain the advantages for someone to use this feature
+
+### What are the disadvantages of the feature?
+- Explain any disadvantages for someone to use this feature
+
+### How will this be implemented or integrated into the O3DE environment?
+- Explain how a developer will integrate this into the codebase of O3DE and provide any specific library or technical stack requirements.
+
+### Are there any alternatives to this feature?
+- Provide any other designs that have been considered. Explain what the impact might be of not doing this.
+- If there is any prior art or approaches with other frameworks in the same domain, explain how they may have solved this problem or implemented this feature.
+
+### How will users learn this feature?
+- Detail how it can be best presented and how it is used as an extension or a standalone tool used with O3DE.
+- Explain if and how it may change how individuals would use the platform and if any documentation must be changed or reorganized.
+- Explain how it would be taught to new and existing O3DE users.
+
+### Are there any open questions?
+- What are some of the open questions and potential scenarios that should be considered?
```

Comment:
That makes sense. To confirm, should every feature in each SIG have RFC? If yes, then I can remove all of this and link it to the RFC instead @Kadino

Reply:
Unclear to me whether a roadmap item will always include one or more RFCs. Not every individual feature needs an RFC.

However duplicating this info from the RFC template will result in prompting duplicated work, which we can try to avoid.

Reply:
@vincent6767  
A roadmap item does not need to have an RFC. (take items on the SIG-Release roadmap - they will often just be issues not RFCs). 
In the case where an RFC exists, we should encourage the writer to link to the RFC, so maybe it's better to have a heading: "Related Links:"   I could also see this new heading being used to link to additional information. for example if some new feature improves the Asset Processor, for example, they could include a link to the documentation for Asset Processor to make it easier for someone to go read more about the underlying system 

I think we should remove lines 24-58. Even if there isn't an RFC, information like this should live in the issue or PR.

Reply:
Got it. I'll remove the 24-58 lines and add a new section "Related Links" to encourage contributors to link additional information.